angular-component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+import moment from 'moment';
+import { CustomerInfoService } from "./services";
+
+@Component({
+  selector: 'my-app',
+  template: `<h1>Customer latest activity date
+  {{ formatDate(latestActivityDate) }}</h1>`,
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+  latestActivityDate: string;
```

Comment:
zmienna nie jest zainicjalizowania, a formatowanie jest odpalane, albo zainicjować to (zeby nie bylo undefined), albo w formatDate dodać sprawdzanie